src/tests/interpolation/test_interpolation_finite_element.cc

```diff
@@ -51,28 +51,38 @@ CASE( "test_interpolation_finite_element" ) {
 
     auto func = []( double x ) -> double { return std::sin( x * M_PI / 180. ); };
 
-    Interpolation interpolation( option::type( "finite-element" ), fs, pointcloud );
+    util::Config config = util::Config( "type", "finite-element" ).set( "maxFractionElemsToTry", 0.4 );
```

Comment:
also here "max_fraction_elems_to_try"

Reply:
should be fixed inthe most recent commit.